products/workers/src/content/learning/using-durable-objects.md

```diff
@@ -347,50 +345,38 @@ $ curl -H "Content-Type: text/plain" https://<worker-name>.<your-namespace>.work
 
-Durable Objects is currently in early beta, and some planned features have not been enabled yet. Many of these limitations will be fixed before Durable Objects becomes generally available.
-
-### Risk of Data Loss
-
-At this time, we are not ready to guarantee that data won't be lost. We don't expect data loss and do maintain regular backups, but bugs are always possible.
-
-For now, if you are storing data in Durable Objects that you can't stand to lose, you must arrange to make backups of that data into some other storage system. Do not rely on Durable Objects for storing production data during the beta period. (This is, of course, always best practice anyway, but it is especially important in the beta.)
+Durable Objects is now generally available, however there are some known limitations.
 
 ### Global Uniqueness
 
 Uniqueness is currently enforced upon starting a new event (such as receiving an HTTP request), and upon accessing storage. After an event is received, if the event takes some time to execute and does not ever access its durable storage, then it is possible that the Durable Object instance may no longer be current, and some other instance of the same object ID will have been created elsewhere. If the event accesses storage at this point, it will receive an exception, but if the event completes without ever accessing storage, it may not ever realize that the object was no longer current.
 
 In particular, a Durable Object may be superseded in this way in the event of a network partition or a software update (including either an update of the Durable Object's class code, or of the Workers system itself).
 
-### Enumerating objects
-
-There is currently no support for generating a list of all existing objects, nor any way to bulk export objects.
-
 ### Development tools
 
-[Wrangler dev](/cli-wrangler/commands#dev) does not currently work with Durable Objects.
+[Wrangler tail](/cli-wrangler/commands#tail) logs from requests that are upgraded to WebSockets are delayed until the WebSocket is closed.  Wrangler tail should not be connected to a Durable Object that you expect will receive heavy volumes of traffic.
 
-[Wrangler tail](/cli-wrangler/commands#tail) does work, but note that logs from requests that are upgraded to WebSockets are delayed until the WebSocket is closed.
-
-The Workers dashboard does not yet support viewing or editing Workers that use modules syntax. It also does not yet display any information about your Durable Objects or allow you to create client bindings to Durable Objects in your Workers.
+The Workers dashboard does not yet support editing Workers that use modules syntax.
 
 ### Object Location
 
-Not all Cloudflare locations support Durable Objects yet, so objects may not be created in exactly the same point-of-presence where they are first requested.
+Not all Cloudflare locations support Durable Objects yet, so objects may not be created in the same point-of-presence where they are first requested.
```

Comment:
`s/Not all Cloudflare locations support Durable Objects yet/Not all Cloudflare locations can host Durable Objects/`

We don't need to promise them everywhere. I also think people may conflate "supporting DOs" with "having access to a DO namespace".